tests/check.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Should be run from root directory of project
+
+# Lint Playbooks
+find . -type f -name "*.yml" ! -path "*meta*" ! -path "*extra*" ! -path "*sg-*" | xargs yamllint
```

Comment:
Can we just use a `yamllint` configuration file?
http://yamllint.readthedocs.io/en/latest/configuration.html#ignoring-paths

I appreciate the `find` though :smile_cat:

Reply:
Also can we move this to a travis/other ci solution?